sys/net/vnet.c

```diff
@@ -244,12 +244,20 @@ vnet_alloc(void)
 	vnet->vnet_data_mem = malloc(VNET_SIZE, M_VNET_DATA, M_WAITOK);
 	memcpy(vnet->vnet_data_mem, (void *)VNET_START, VNET_BYTES);
 
+#ifndef __CHERI_PURE_CAPABILITY__
```

Comment:
I usually prefer using #ifdef __CHERI_PURE_CAPABILITY__ and leaving the non-purecap case in an #else, but in this case the comment flows better in the order used here.

Reply:
I tend to like adding an extra paragraph to the comment and hoisting it above the #if(def). It also makes it easier to compare the two branches (though in this case that's not so much of an issue being very simple one-line statements).